src/Adjudications/AdjudicationDetail/Hearing.js

```diff
@@ -1,48 +1,54 @@
 import GridRow from '@govuk-react/grid-row'
 import GridCol from '@govuk-react/grid-col'
+import { H3 } from '@govuk-react/heading/lib/presets'
 import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
-import { H3 } from '@govuk-react/heading/lib/presets'
-import { LabelAndValue } from './AdjudicationDetail.styles'
+import { LabelAndValue, Location, Section } from './AdjudicationDetail.styles'
 
 export const Hearing = ({ hearing }) => (
   <>
-    <H3>Hearing Details</H3>
-    {hearing ? (
-      <>
-        <GridRow>
-          <GridCol>
-            <LabelAndValue label="Type" value={hearing.hearingType} />
-          </GridCol>
-          <GridCol>
-            <LabelAndValue label="Date" value={hearing.hearingTime} />
-          </GridCol>
-          <GridCol>
-            <LabelAndValue label="Location" value={hearing.location} />
-          </GridCol>
-          <GridCol>
-            <LabelAndValue label="Heard by" value={hearing.heardByName} />
-          </GridCol>
-          <GridCol>
-            <LabelAndValue label="Other representatives" value={hearing.otherRepresentatives} />
-          </GridCol>
-          <GridCol>
-            <LabelAndValue label="Comment" value={hearing.comment} />
-          </GridCol>
-        </GridRow>
-      </>
-    ) : (
-      <p>No information available</p>
-    )}
-    <hr />
+    <Section>
+      <H3>Hearing details</H3>
+      {hearing ? (
+        <>
+          <GridRow>
+            <GridCol setWidth="20%">
```

Comment:
would `setDesktopWidth` suit these better? Just wondering if they'd look a bit squashed on smaller screens.

Reply:
They collapse in mobile view so I guess it's ok?:

<img width="322" alt="Screenshot 2019-05-31 at 16 23 49" src="https://user-images.githubusercontent.com/1517745/58716277-964b7d80-83c0-11e9-9642-c80c8d5c3b4c.png">